[2024-07-18T07:57:34.187+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-18T07:57:34.202+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_id_3.log_database_connection manual__2024-07-18T07:57:33.070381+00:00 [queued]>
[2024-07-18T07:57:34.207+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_id_3.log_database_connection manual__2024-07-18T07:57:33.070381+00:00 [queued]>
[2024-07-18T07:57:34.207+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-18T07:57:34.213+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): log_database_connection> on 2024-07-18 07:57:33.070381+00:00
[2024-07-18T07:57:34.217+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=69) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-18T07:57:34.218+0000] {standard_task_runner.py:63} INFO - Started process 73 to run task
[2024-07-18T07:57:34.218+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dag_id_3', 'log_database_connection', 'manual__2024-07-18T07:57:33.070381+00:00', '--job-id', '1553', '--raw', '--subdir', 'DAGS_FOLDER/dag_id_3.py', '--cfg-path', '/tmp/tmpryockhob']
[2024-07-18T07:57:34.220+0000] {standard_task_runner.py:91} INFO - Job 1553: Subtask log_database_connection
[2024-07-18T07:57:34.264+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_id_3.log_database_connection manual__2024-07-18T07:57:33.070381+00:00 [running]> on host 4d13026947d2
[2024-07-18T07:57:34.337+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_id_3' AIRFLOW_CTX_TASK_ID='log_database_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-07-18T07:57:33.070381+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-18T07:57:33.070381+00:00'
[2024-07-18T07:57:34.338+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-18T07:57:34.338+0000] {resources.py:20} INFO - dag_id_3 start processing tables in database:airflow
[2024-07-18T07:57:34.339+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-18T07:57:34.339+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-18T07:57:34.353+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_id_3, task_id=log_database_connection, run_id=manual__2024-07-18T07:57:33.070381+00:00, execution_date=20240718T075733, start_date=20240718T075734, end_date=20240718T075734
[2024-07-18T07:57:34.398+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-18T07:57:34.417+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-18T07:57:34.418+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
